docs/source/llms/sentence-transformers/tutorials/quickstart/sentence-transformers-quickstart.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction to Sentence Transformers and MLflow\n",
+    "\n",
+    "Welcome to our tutorial on leveraging **Sentence Transformers** with **MLflow** for advanced natural language processing and model management.\n",
+    "\n",
+    "### Learning Objectives\n",
+    "\n",
+    "- Set up a pipeline for sentence embeddings with `sentence-transformers`.\n",
+    "- Log models and configurations using MLflow.\n",
+    "- Understand and apply model signatures in MLflow to `sentence-transformers`.\n",
+    "- Deploy and use models for inference with MLflow's features.\n",
+    "\n",
+    "<details>\n",
+    "    <summary style=\"cursor: pointer; display: flex; align-items: center;\">\n",
+    "        <span style=\"margin-right: 10px;\">&#x25BA;</span>\n",
+    "        <span>Expand to learn more about Sentence Transformers and MLflow integration.</span>\n",
+    "    </summary>\n",
+    "    <br/>\n",
+    "    <div>\n",
+    "        <h4>What are Sentence Transformers?</h4>\n",
+    "        <p>Sentence Transformers, an extension of the Hugging Face Transformers library, are designed for generating semantically rich sentence embeddings. They utilize models like BERT and RoBERTa, fine-tuned for tasks such as semantic search and text clustering, producing high-quality sentence-level embeddings.</p>\n",
+    "    </div>\n",
+    "    <div>\n",
+    "        <h4>Benefits of Integrating MLflow with Sentence Transformers</h4>\n",
+    "        <p>Combining MLflow with Sentence Transformers enhances NLP projects by:</p>\n",
+    "        <ul>\n",
+    "            <li>Streamlining experiment management and logging.</li>\n",
+    "            <li>Offering better control over model versions and configurations.</li>\n",
+    "            <li>Ensuring reproducibility of results and model predictions.</li>\n",
+    "            <li>Simplifying the deployment process in production environments.</li>\n",
+    "        </ul>\n",
+    "        <p>This integration empowers efficient tracking, management, and deployment of NLP applications.</p>\n",
+    "    </div>\n",
+    "</details>\n",
+    "<br/>\n",
+    "Join us in exploring the integration of Sentence Transformers with MLflow for efficient NLP model management.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting Up the Environment for Sentence Embedding\n",
+    "\n",
+    "Begin your journey with Sentence Transformers and MLflow by establishing the core working environment.\n",
+    "\n",
+    "#### Key Steps for Initialization\n",
+    "\n",
+    "- Import necessary libraries: `SentenceTransformer` and `mlflow`.\n",
+    "- Initialize the `\"all-MiniLM-L6-v2\"` Sentence Transformer model.\n",
+    "\n",
+    "<details>\n",
```

Comment:
This is a very odd bug in nbsphinx and it's due to the conversion that they do (markdown -> pandoc convert to .rst -> convert to html) which doesn't have full support for all html tags. specifically, the `<summary>` tag is not supported and gets converted to a `<p>` instead. 
I removed the summary content, restructured the `<details>` elements, and added notes at the top of the notebooks to expand those sections if a reader wants to learn more. 
We don't have many options here aside from creating custom CSS classes and overriding the DOM behavior with JS (which I don't want to do).

Reply:
I see, sadly seems like nbshpinx has many limitations... Also agree with that we don't want to spend effort with custom JS/CSS logic, wish it would be fixed when migrating to MD framework.